targets/lenovo-x1-carbon.nix

```diff
@@ -182,8 +187,11 @@
           powerControl = pkgs.callPackage powerControlPkgPath {};
         in [
           {
+            # The SPKI fingerprint is calculated like this:
+            # $ openssl x509 -noout -in mitmproxy-ca-cert.pem -pubkey | openssl asn1parse -noout -inform pem -out public.key
+            # $ openssl dgst -sha256 -binary public.key | openssl enc -base64
             name = "chromium";
-            path = "${pkgs.openssh}/bin/ssh -i ${sshKeyPath} -o StrictHostKeyChecking=no chromium-vm.ghaf run-waypipe chromium --enable-features=UseOzonePlatform --ozone-platform=wayland";
+            path = "${pkgs.openssh}/bin/ssh -i ${sshKeyPath} -o StrictHostKeyChecking=no chromium-vm.ghaf run-waypipe chromium --enable-features=UseOzonePlatform --ozone-platform=wayland --user-data-dir=~/.config/chromium/Default --ignore-certificate-errors-spki-list=Bq49YmAq1CG6FuBzp8nsyRXumW7Dmkp7QQ/F82azxGU=";
```

Comment:
This should also be guarded, doesn't break it but gives the warning

Reply:
Guard present in [b500944](https://github.com/tiiuae/ghaf/commit/b500944e5baef64e8efefa4b2e5fbbedcdbbbce5)